staubli_rx160_gazebo/urdf/rx160.xacro

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<robot name="staubli_rx160_gazebo" xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- RX160 -->
+  <xacro:include filename="$(find staubli_rx160_gazebo)/urdf/rx160_macro.xacro"/>	
+  
+  <xacro:staubli_rx160_robot prefix=""/>
```

Comment:
Depending on how we decide to name the `normal model + gazebo elements` xacro macro, we might actually use the `_gazebo` suffix here, instead of `_robot` as you did.

This file `rx160.xacro` is actually only meant as a *top-level* xacro (which cannot be included in other xacros), and the `staubli_rx160_gazebo/urdf/rx160_macro.xacro` defines the actual xacro macro, so it would make sense to include some reference to Gazebo in the macro name, and not in the 'robot name' of this top-level xacro.